crates/configuration/src/version3/mod.rs

```diff
@@ -1,9 +1,9 @@
 //! Internal Configuration and state for our connector.
 
-mod comparison;
+pub(crate) mod comparison;
```

Comment:
These need to be more visible than before so that `upgrade_from_v3` can access them.